task-3/routers/user.ts

```diff
@@ -0,0 +1,77 @@
+import express, { Request } from "express";
+import { ValidatedRequest } from "express-joi-validation";
+
+import { UserService } from "../services/user";
+import {
+  validator,
+  postSchema,
+  putSchema,
+  CreateUserSchema,
+} from "../validation";
+
+const router = express.Router();
+
+router.get(
+  "/suggested",
+  async (
+    req: Request<unknown, unknown, unknown, { login: string; limit: number }>,
+    res
+  ) => {
+    const { login, limit } = req.query;
+    try {
+      const users = UserService.getSuggestedUsers(login, limit);
+      res.send(users);
+    } catch (err) {
+      res.status(404);
+    }
+  }
+);
+
+router.get("/:id", async (req, res) => {
+  try {
+    const user = await UserService.findAllUsers(req.params.id);
+    res.send(user);
+  } catch (err) {
+    res.status(404);
```

Comment:
Consider using [http-status-codes](https://www.npmjs.com/package/http-status-codes) npm package. It could help you improve code readability and avoid using wrong status codes in case your app grows.